test/stdlib/Enumerable_test.rb

```diff
@@ -83,11 +83,6 @@ def test_each_entry
     enumerable.each_entry { |x| x }
   end
 
-  def test_zip
```

Comment:
I didn't understand why these test methods existed. According to the following PRs, I guess none of the runtime tests are executed intentionally:

* https://github.com/ruby/rbs/pull/320
* https://github.com/ruby/rbs/pull/405